data/seeds/03-category.js

```diff
@@ -0,0 +1,54 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex("Category").truncate()
+    .then(function () {
+      
+      let num = 1;
+      // Inserts seed entries
+      return knex("Category").insert([
+        {id: num++, category: "Animal Products", loc_id: Math.floor(Math.random()*54)+1},
```

Comment:
I don't categories themselves need a location id. Just specific items being sold. Categories should persist for all locations

Reply:
Also these foreign keys seem to be entirely random and not reference an actual location

Reply:
In order to get the average price on an item based on location, I had to connect category to location.  The request URL looks like this:

/api/location/id_of_the_location/category/id_of_the_category/items/avg

Looks odd, but it works.

Reply:
The foreign keys references the location ids.  There are 54 countries so I randomly generated ids.  This is dummies info for testing.